.github/workflows/ISSUE_TEMPLATE/bug_report.yaml

```diff
@@ -0,0 +1,75 @@
+name: Bug Report
+description: Create a report to help us reproduce and fix a bug
+labels:
+  - "type: bug"
+  - "status: triage"
+
+body:
+  - type: markdown
+    attributes:
+      value: >
+        #### Thank you for contributing! Before reporting a bug, please make sure it has no duplicate and/or has already been addressed by searching through [the existing and past issues](https://github.com/BerkeleyLearnVerify/Scenic/issues)
+
+  - type: textarea
+    attributes:
+      label: System Details
+      description: |
+        Please provide the following system information to help us diagnose the bug.
+        1. Python Version
+        2. Scenic Version
+        3. Operating System / Platform
+        4. Simulator Version (Optional, only applies if bug is simular specific)
+
+      placeholder: |
+        1. Python Version: `python --version`
+        2. Scenic Version: `scenic --version`
+        3. Operating System / Platform: get from system preferences
+        4. Simulator Version: simulator version
+
+        Example:
+        1. Python Version: Python 3.11.7
+        2. Scenic Version: Scenic 3.0.0b2
+        3. Operating System / Platform: Apple M2 Pro macOS Sonoma 14.2.1
+        4. Simulator Version: CARLA 0.9.14
+      validations:
+        required: true
+
+  - type: textarea
+    attributes:
+      label: Detailed Description
+      description: |
+        Please provide a clear and concise description of what the bug is and paste the error log below. Please rerun your simulation error with the `-b` parameter for a full stack trace.
+        It helps improving readability if the error log is wrapped in ```triple quotes blocks```.
+      placeholder: |
+        A clear and concise description of what the bug is. You can rerun your Scenic simulation with `-b` to get a full stack trace.
+
+        ```
+        # error full stack trace
+        ```
+      validations:
+        required: true
+
+  - type: textarea
+    attributes:
+      label: Steps To Reproduce
+      description: |
+        Please provide a minimal example to help us reproduce the bug. Code should be wrapped with ```triple quotes blocks``` to improve readability. If the code is too long, please attach as a file or create and link a public gist: https://gist.github.com.
+        Related data files (images, videos, etc.) should be attached below as well. If the data files are too big, feel free to upload them to an online drive, share them, and put the link below.
```

Comment:
I want to recommend a compressed `.zip` or `.tar.gz` file, but I'm not sure if those are secure in general from untrusted sources (We don't want to get zipbombed). @dfremont any thoughts?

Reply:
I would rather discourage the uploading of files entirely -- we don't want to be in the position of taking somebody's big codebase and trying to figure out what's going on. I get that someone might need to upload a problematic mesh for example, but that should be an unusual case. What about replacing everything from "If the code is too long" onward with something like "If it is not possible to reproduce the bug with a short self-contained Scenic or Python file, for example if a specific mesh is required, you can attach any required files below or include a link to them."?

Reply:
Agreed, let me clarify the language here to ensure we don't get users submitting unnecessary files!